lib/OpenCloud/Database/Resource/Backup.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright 2012-2014 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCloud\Database\Resource;
+
+use OpenCloud\Common\Lang;
+use OpenCloud\Common\Exceptions;
+use OpenCloud\Common\Resource\PersistentResource;
+use OpenCloud\Database\Service;
+
+/**
+ * This class represents a Backup
+ */
+class Backup extends PersistentResource
+{
+    public $id;
+    public $name;
+    public $description;
+    public $created;
+    public $datastore;
+    public $updated;
+    public $instance;
+    public $instance_id;
+    public $locationRef;
+
+    protected static $json_name = 'backup';
+    protected static $url_resource = 'backups';
+
+    protected $createKeys = array(
+        'name',
+        'instance',
+        'description'
+    );
+
+    protected $associatedResources = array(
+        'instance_id' => 'Instance'
```

Comment:
Shouldn't the key be `instance`, since it represents an instance object? `instance_id` will be a string.